src/main/java/org/sunbird/workallocation/util/LastLoginConstants.java

```diff
@@ -0,0 +1,29 @@
+package org.sunbird.workallocation.util;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class LastLoginConstants {
```

Comment:
Move all these variables into existing Constants file... we don't require additional file for this.